<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>698</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Telemetry</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>561</width>
         <height>201</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLabel" name="state_label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>State</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="grbl_status">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>GRBL Status</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QLCDNumber" name="position_y">
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="value" stdset="0">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLCDNumber" name="position_x">
          <property name="digitCount">
           <number>10</number>
          </property>
          <property name="value" stdset="0">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="state_value">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Grid Parameters</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>551</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0">
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="az_increment">
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>15.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Az increment</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Start Az</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>End Az</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Start Alt</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="end_az_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QDoubleSpinBox" name="end_alt_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>90.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="start_az_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>0.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>End Alt</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="start_alt_spinbox">
          <property name="minimum">
           <double>0.000000000000000</double>
          </property>
          <property name="maximum">
           <double>360.000000000000000</double>
          </property>
          <property name="value">
           <double>60.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDoubleSpinBox" name="alt_increment">
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
          <property name="value">
           <double>10.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Alt increment</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Directional commands</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>351</width>
         <height>211</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="home_x_button">
          <property name="text">
           <string>X</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/home.svg</normaloff>:/Images/Images/home.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="home_y_button">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/home.svg</normaloff>:/Images/Images/home.svg</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="search_grid_button">
          <property name="text">
           <string>Search Grid</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/Images/Images/home.svg</normaloff>:/Images/Images/home.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
